user/papers/patches/tests.patch

```diff
@@ -0,0 +1,18 @@
+--- a/data/meson.build	2024-06-17 07:49:52.000000000 -0300
++++ b/data/meson.build	2024-09-11 00:20:22.936416648 -0300
+@@ -59,15 +59,3 @@
+   install: true,
+   install_dir: join_paths(pps_datadir, 'metainfo'),
+ )
+-
+-# Validate Appdata
+-if get_option('tests')
+-  appstreamcli = find_program('appstreamcli', required: true)
+-  if (appstreamcli.found())
+-    test('validate-appdata',
+-         appstreamcli,
+-         args: ['validate', '--strict', '--pedantic', '--explain', metainfo_file],
```

Comment:
you can probably add only `--nonet` to these args and it will work :)

Reply:
omg true